src/pages/Roadmap.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import styled from 'styled-components';
+import {Page} from '../components';
+import SecondaryNavbar from '../components/Project/SecondaryNavbar';
+import AragonClient from '../components/Project/AragonClient';
+import AragonNetwork from '../components/Project/AragonNetwork';
+import FeedbackModule from '../components/Project/FeedbackModule';
+import Section from '../components/General/Section';
+import backgroundImage from '../components/Project/assets/roadmap-background.png';
+import {Button} from '@aragon/ui';
+
+class Roadmap extends React.Component {
+  state = {active: 'left'};
+
+  render() {
+    const {active} = this.state;
+    return (
+      <Page path="/project/roadmap">
+        <SecondaryNavbar page="roadmap" />
+        <Hero>
+          <Container>
+            <h2>Roadmap</h2>
+            <h4>
+              We want to build the best ecosystem for decentralized
```

Comment:
The color set for this text with the current background made it hard for me to read, I tried white and black and black provided a better contrast in my opinion. Just leaving it here in case you had thought about it.

Reply:
Thanks for the feedback! I will talk to Adri about it. For now I am leaving it this way and may be will update in a second iteration :)